### PR TITLE
Make construction cost color slightly lighter on central and northern maps

### PR DESCRIPTION
**Pull request by crudelios**
Since I'm color blind, I had trouble reading the construction cost on the northern/central maps as for me it looked almost the same as the terrain color.

This improves things without changing the overall color of the construction cost. It's slightly lighter on northern and central maps, and the same color on desert maps.

However, short of adding a white box like in the tooltips, some color blind people with different types of color blindness will still have trouble reading the cost. Hopefully that's rare enough that it won't matter.

### CODE REVIEW

**Comment by m-mohr** [comment]
Could you share a screenshot, please? :-)

**Comment by bvschaik** [comment]
Top - original, bottom - new.
build-color

File: src/widget/city.c [hunk 1]
```diff
@@ -72,7 +73,8 @@ void widget_city_draw_construction_cost(void)
     set_city_clip_rectangle();
     color_t color;
     if (cost <= city_finance_treasury()) {
-        color = COLOR_ORANGE;
+        // Color blind friendly
+        color = (scenario_property_climate() == CLIMATE_DESERT) ? COLOR_ORANGE : COLOR_ORANGE_LIGHT;
```
Comment (bvschaik) [inline src/widget/city.c]: Parentheses are not necessary

Comment (m-mohr) [inline src/widget/city.c]: By the way, do you all know that you can make commitable suggestions here?

For example:
```suggestion
        color = scenario_property_climate() == CLIMATE_DESERT ? COLOR_ORANGE : COLOR_ORANGE_LIGHT;
```

Comment (crudelios) [reply src/widget/city.c]: Nice! Thanks! Saves some work.

**Pull request merged**